### ISSUE TITLE
anyboday have pytorch verision of this method??

### ISSUE THREAD

**Issue by NomiMalik0207** [MSiam/TFSegmentation#54]

**Comment by MSiam** [comment]
Nope. But you can check this [repo](https://github.com/meetshah1995/pytorch-semseg) its very useful that has multiple segmentation architectures but not aimed at real-time performance. You can use it as a start and add yourself shufflenet and MobileNet.
